worker/uniter/modes.go

```diff
@@ -330,7 +368,7 @@ func ModeHookError(u *Uniter) (next Mode, err error) {
 			}
 			return ModeContinue, nil
 		case actionId := <-u.f.ActionEvents():
-			if err := u.runOperation(newActionOp(actionId)); err != nil {
+			if err := u.runOperation(newActionOp(actionId), fmt.Sprintf("running action %q", actionId)); err != nil {
```

Comment:
I really do worry about the degree of spam we're subjecting the api server to here -- if a user cares that we're executing X action, that should be recorded against the action itself, shouldn't it?